src/common-db/src/main/java/ee/ria/xroad/common/db/CustomPostgreSQLDialect.java

```diff
@@ -30,5 +30,6 @@
 /**
  * @deprecated To be removed
  */
+@Deprecated
 public class CustomPostgreSQLDialect extends PostgreSQL9Dialect {
```

Comment:
Could this whole file be removed, if it is deprecated?   
At least it looks empty.

Reply:
It is possibly referenced in some scripts, not in scope of this ticket. Annotation added to silence a compiler warning.